src/ProgressMeter/ProgressMeter.jl

```diff
@@ -31,24 +31,24 @@ Print progress bar to `io`.
 - `barglyphs::BarGlyphs`
 - `tfirst::Float64`
 - `desc`: description to be printed at left side of progress bar.
-- `progress::Real`: a number between 0 and 1 or a `NaN`.
+- `progress::Union{Float64, Nothing}`: a number between 0 and 1 or `nothing`.
 - `eta_seconds::Real`: ETA in seconds
 """
 function printprogress(
     io::IO,
     barglyphs::BarGlyphs,
     tfirst::Float64,
     desc,
-    progress::Real,
+    progress::Union{Float64, Nothing},
```

Comment:
```suggestion
    progress,
```
I suggest we just remove the type annotation here, as it's not needed for dispatch.

Reply:
Thanks for pointing this oversight out! Fixed in https://github.com/c42f/TerminalLoggers.jl/pull/33/commits/79cc63789ee90ea4e70583dd5eea3c4b7be99f74